.buildkite/pipeline.yml

```diff
@@ -11,6 +11,12 @@
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
 
+env:
+  # Firectl tests will run against this version of firecracker:
+  FIRECRACKER_VERSION: 0.19.0
+  # Firecracker binaries will be installed and run from this directory:
+  BINDIR: "${HOME}/${BUILDKITE_BUILD_NUMBER}-bin"
```

Comment:
You could copy it into the working directory for the test, which will get automatically cleaned up when the test ends.  That would make this change a lot simpler as you wouldn't need to introduce a new explicit cleanup step.

Reply:
I've got a deep-seated aversion to adding '.' to PATH, which is why I avoided that.

Reply:
What about moving it into `$(pwd)/bin` and adding that to `$PATH`?  That'd still get you the automatic cleanup provided by BuildKite and avoid adding '.'.

Reply:
the `/bin` is irrelevant. `$PWD` will get the same effect. I am OK with that. (`$PWD` will get expanded when it's added to `$PATH`, while `.` is always the _current_ working directory.)

Reply:
I've updated this to use a subdirectory of `$PWD`. Please have a look @samuelkarp